src/views/List.jsx

```diff
@@ -1,6 +1,13 @@
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	console.log(data);
+
+	// Q: const or let?
+	const listItem = data.map((item) => {
+		return <ListItem key={item.id} name={item.name} />;
+	});
+
```

Comment:
I believe `const` is fine in this case. `let` is used if the value of `listItem` is going to change. In this case, it's only going to be a constant value, which is returning the `ListItem` component.

Reply:
Agree with Ashelle!